state/txindex/kv/kv.go

```diff
@@ -749,25 +750,30 @@ func (txi *TxIndex) matchRange(
 		panic(err)
 	}
 	defer it.Close()
+	bigIntValue := new(big.Int)
 
 LOOP:
 	for ; it.Valid(); it.Next() {
-		if !isTagKey(it.Key()) {
+		// TODO: We need to make a function for getting it.Key() as a byte slice with no copies.
```

Comment:
Would you mind opening an issue for this?

Reply:
Sure! Do you want me to delete the comment as well?

Reply:
cometbft-db issue opened: https://github.com/cometbft/cometbft-db/issues/156

Reply:
> Sure! Do you want me to delete the comment as well?

no